src/portal-api.ts

```diff
@@ -465,8 +465,13 @@ export const anonymousPortalData = (preview: boolean) => {
   } else {
     runKey = queryValue("runKey");
     if (!runKey) {
-      runKey = uuidv4();
-      setQueryValue("runKey", runKey);
+      if (isOfflineHost()) {
+        runKey = "offline";
```

Comment:
Perhaps this is nit picky, but wouldn't it be better if you change changed line 462 to 
`if (preview || isOfflineHost()) {`

But perhaps you wanted the runKey of `offline` to be overridden if some passed a query parameter?

Reply:
@scytacki  I went back and forth on this.  I had a helper like `const createRunKey = (fallback: string) => isOfflineHost() ? "offline" : fallback;` and I used that on lines 462 and 468 but then looking back at the call chain I thought if we have an explicit `preview` url parameter we shouldn't pollute any offline data we have saved.

Reply:
Sounds fine to me.